ray_data_eval/libsolver/src/solver/environment.rs

```diff
@@ -270,11 +331,29 @@ impl Executor {
             if task.remaining_ticks > 0 {
                 self.timeline.push_str(&task.spec.id);
                 self.timeline.push_str("  ");
+
+                if task.remaining_ticks == task.spec.duration as i32 {
+                    let dur_time = (task.spec.duration as u64) * MICROSECS_PER_TICK;
+                    let event = TraceEvent::new(
+                        "task".to_string(),
+                        task.spec.id.clone(),
+                        "1".to_string(),
+                        self.name.clone(),
+                        self.current_time,
+                        dur_time,
+                        "X".to_string(),
+                        self.color_map[&task.spec.id].clone(),
```

Comment:
@ronyw7 I got a crash here when my task name wasn't one of `PCIT`. Can you add a default color here?

Reply:
Yep! Resolved in #58